packages/content/src/requests/items/deleteAttribute/DeleteAttributeAcceptResponseItem.ts

```diff
@@ -0,0 +1,27 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { CoreDate, ICoreDate } from "@nmshd/transport";
+import { AcceptResponseItem, AcceptResponseItemJSON, IAcceptResponseItem } from "../../response";
+
+export interface DeleteAttributeAcceptResponseItemJSON extends AcceptResponseItemJSON {
+    "@type": "DeleteAttributeAcceptResponseItem";
+    deletionDate: string;
+}
+
+export interface IDeleteAttributeAcceptResponseItem extends IAcceptResponseItem {
+    deletionDate: ICoreDate;
+}
+
+@type("DeleteAttributeAcceptResponseItem")
+export class DeleteAttributeAcceptResponseItem extends AcceptResponseItem implements IDeleteAttributeAcceptResponseItem {
+    @serialize()
+    @validate()
+    public deletionDate: CoreDate;
+
+    public static override from(value: IDeleteAttributeAcceptResponseItem | Omit<DeleteAttributeAcceptResponseItemJSON, "@type">): DeleteAttributeAcceptResponseItem {
```

Comment:
you should still allow to add `@type`, so `IX | Omit<XJSON, "@type"> | XJSON` would be better here